template.yaml

```
@@ -114,4 +114,4 @@ Resources:
         Timer:
           Type: Schedule
           Properties:
-              Schedule: cron(0 0 1 * ? *)
+              Schedule: cron(0 0 * * * *)
```

Looking at the [docs](https://docs.aws.amazon.com/lambda/latest/dg/services-cloudwatchevents-expressions.html) I think you need to add a ? for the day to specify it's every day.

```suggestion
              Schedule: cron(0 0 * ? * *)
```